tests/migration/online_migration/register_ltss.pm

```diff
@@ -0,0 +1,33 @@
+# SUSE's openQA tests
+#
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+#
+# Summary: Add the ltss extension only
+# For instance, this module is needed when we test a rolling upgrade
+# migration from a LTSS version to another one LTSS version.
+# Maintainer: Julien Adamek <jadamek@suse.com>
+
+use base "opensusebasetest";
+use strict;
+use warnings;
+use testapi;
+use registration qw(add_suseconnect_product cleanup_registration);
+
+sub run {
+    my ($self) = @_;
+    $self->select_serial_terminal;
+
+    my @scc_addons = split(/,/, get_var('SCC_ADDONS', ''));
+    if (grep $_ eq 'ltss', @scc_addons) {
+        my $os_sp_version = get_var("HDDVERSION");
+        $os_sp_version =~ s/-/_/g;
```

Comment:
Could be wrote in one line I think: `my $os_sp_version = get_var("HDDVERSION") =~ s/-/_/g;`

Reply:
I tested this and unfortunately, it doesn't work. 
If I remember correctly, I've already tested it when I pushed this code for the first time.